[2022-03-17 03:22:14,470] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: testing_dag.ingest 2019-07-02T06:00:00+00:00 [queued]>
[2022-03-17 03:22:14,514] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: testing_dag.ingest 2019-07-02T06:00:00+00:00 [queued]>
[2022-03-17 03:22:14,516] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 03:22:14,519] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-03-17 03:22:14,521] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 03:22:14,547] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): ingest> on 2019-07-02T06:00:00+00:00
[2022-03-17 03:22:14,554] {standard_task_runner.py:52} INFO - Started process 1022 to run task
[2022-03-17 03:22:14,563] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'testing_dag', 'ingest', '2019-07-02T06:00:00+00:00', '--job-id', '23', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/testing_dag.py', '--cfg-path', '/tmp/tmp54wycu3i', '--error-file', '/tmp/tmp242s5c8o']
[2022-03-17 03:22:14,569] {standard_task_runner.py:77} INFO - Job 23: Subtask ingest
[2022-03-17 03:22:14,681] {logging_mixin.py:104} INFO - Running <TaskInstance: testing_dag.ingest 2019-07-02T06:00:00+00:00 [running]> on host 509d5aa4e827
[2022-03-17 03:22:14,789] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=testing_dag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2019-07-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-07-02T06:00:00+00:00
[2022-03-17 03:22:14,792] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-03-17 03:22:14,797] {bash.py:158} INFO - Running command: ls /opt/airflow
[2022-03-17 03:22:14,812] {bash.py:169} INFO - Output:
[2022-03-17 03:22:14,889] {bash.py:173} INFO - airflow.cfg
[2022-03-17 03:22:14,892] {bash.py:173} INFO - dags
[2022-03-17 03:22:14,894] {bash.py:173} INFO - google_credentials.json
[2022-03-17 03:22:14,896] {bash.py:173} INFO - logs
[2022-03-17 03:22:14,897] {bash.py:173} INFO - output_2019-01.csv
[2022-03-17 03:22:14,901] {bash.py:173} INFO - output_2019-02.csv
[2022-03-17 03:22:14,904] {bash.py:173} INFO - output_2019-03.csv
[2022-03-17 03:22:14,905] {bash.py:173} INFO - output_2019-04.csv
[2022-03-17 03:22:14,906] {bash.py:173} INFO - output_2019-05.csv
[2022-03-17 03:22:14,908] {bash.py:173} INFO - output_2019-06.csv
[2022-03-17 03:22:14,910] {bash.py:173} INFO - output_2019-07.csv
[2022-03-17 03:22:14,911] {bash.py:173} INFO - output_2019-08.csv
[2022-03-17 03:22:14,912] {bash.py:173} INFO - output_2019-09.csv
[2022-03-17 03:22:14,914] {bash.py:173} INFO - plugins
[2022-03-17 03:22:14,916] {bash.py:173} INFO - unittests.cfg
[2022-03-17 03:22:14,917] {bash.py:173} INFO - webserver_config.py
[2022-03-17 03:22:14,918] {bash.py:177} INFO - Command exited with return code 0
[2022-03-17 03:22:14,972] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=testing_dag, task_id=ingest, execution_date=20190702T060000, start_date=20220317T032214, end_date=20220317T032214
[2022-03-17 03:22:15,037] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-17 03:22:15,056] {local_task_job.py:146} INFO - Task exited with return code 0
